nix_update/__init__.py

```diff
@@ -23,6 +23,9 @@ def parse_args() -> Options:
     parser.add_argument(
         "--test", action="store_true", help="Run package's `passthru.tests`"
     )
+    parser.add_argument(
+        "--review", action="store_true", help="Run `nixpkgs-review wip`"
```

Comment:
Looks good. Please also add to the readme.

Reply:
lol, I always forget the dang README xD